media/videostitcher/create_cdn_key_akamai.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package videostitcher
+
+// [START videostitcher_create_cdn_key_akamai]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	stitcher "cloud.google.com/go/video/stitcher/apiv1"
+	"cloud.google.com/go/video/stitcher/apiv1/stitcherpb"
+)
+
+// createCdnKeyAkamai creates an Akamai CDN key. A CDN key is used to retrieve
+// protected media.
+func createCdnKeyAkamai(w io.Writer, projectID, cdnKeyID, hostname, akamaiTokenKey string) error {
+	// projectID := "my-project-id"
+	// cdnKeyID := "my-cdn-key"
+	// hostname := "cdn.example.com"
+	// akamaiTokenKey := "VGhpcyBpcyBhIHRlc3Qgc3RyaW5nLg=="
+	location := "us-central1"
+	ctx := context.Background()
+	client, err := stitcher.NewVideoStitcherClient(ctx)
+	if err != nil {
+		return fmt.Errorf("stitcher.NewVideoStitcherClient: %v", err)
+	}
+	defer client.Close()
+
+	var req = &stitcherpb.CreateCdnKeyRequest{
```

Comment:
No need for `var` here. Use the walrus operator:

```go
req := &stitcherpb.CreateCdnKeyRequest{ ... }
```

Reply:
Done